README.md

```diff
@@ -88,31 +88,49 @@ The easiest way to build vcf-validator is using the Docker image provided with t
 
 Executables will be created in the `build/bin` subfolder.
 
+
 ### Linux
 
+The build has been tested on the following distros:
+* Ubuntu
+* Arch
```

Comment:
I'm somewhat reluctant to state this, because in the future we will only test the build **for sure** under Travis' environment.

Reply:
hmm sounds reasonable